std/math.d

```diff
@@ -4515,20 +4515,34 @@ public:
     version (IeeeFlagsSupport)
     {
 
-     /// The result cannot be represented exactly, so rounding occurred.
-     /// (example: x = sin(0.1); )
+     /**
+      * The result cannot be represented exactly, so rounding occurred.
+      * Example: `x = sin(0.1);`
+      */
```

Comment:
Is this just to make comments all use the same formatting in phobos or is this syntax now considered improper? I prefer to use `///` for multiple lines as it is more consistent and easier to use than `/** * */`.